src/traces/parcats/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+* Copyright 2012-2018, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Parcats = {};
+
+Parcats.attributes = require('./attributes');
+Parcats.supplyDefaults = require('./defaults');
+Parcats.calc = require('./calc');
+Parcats.plot = require('./plot');
+Parcats.colorbar = {
+    container: 'line',
+    min: 'cmin',
+    max: 'cmax'
+};
+
+Parcats.moduleType = 'trace';
+Parcats.name = 'parcats';
+Parcats.basePlotModule = require('./base_plot');
+Parcats.categories = ['noOpacity', 'markerColorscale'];
```

Comment:
:zap: `markerColorscale` which is now obsolete (from https://github.com/plotly/plotly.js/pull/2681/).

Reply:
Done in https://github.com/plotly/plotly.js/pull/2963/commits/b1ef6c94789fb10a3a57dba29a3c0170a9d9553c